### PR TITLE
New version: ownCloud.ownCloudDesktop version 5.0.0.12278

### PR DESCRIPTION
**Pull request by SpecterShell**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/122090)

### CODE REVIEW

**Comment by SpecterShell** [comment]
@stephengillie Could you add a waiver for this PR, according to <https://github.com/microsoft/winget-pkgs/pull/120031>?

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978334967 
2023-10-23 14:02:32.589 [CLI ] ShellExecute installer failed: 3010 
MSI (s) (7C:58) [14:02:32:105]: Windows Installer installed the product. Product Name: ownCloud. Product Version: 5.0.0.12278. Product Language: 1033. Manufacturer: ownCloud GmbH. Installation success or error status: 0.

**Comment by stephengillie** [comment]
Digging deeper, it looks like `3010` indicates that a reboot is required.
> MSI (s) (7C:58) [14:02:32:105]: Windows Installer requires a system restart. Product Name: ownCloud. Product Version: 5.0.0.12278. Product Language: 1033. Manufacturer: ownCloud GmbH. Type of System Restart: 2. Reason for Restart: 2.
> MSI (s) (7C:58) [14:02:32:105]: Product: ownCloud. Restart required. The installation or update for the product required a restart for all changes to take effect.  The restart was deferred to a later time.

File: manifests/o/ownCloud/ownCloudDesktop/5.0.0.12278/ownCloud.ownCloudDesktop.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ownCloud.ownCloudDesktop
+PackageVersion: 5.0.0.12278
+InstallerType: wix
+Scope: machine
+InstallerSwitches:
+  InstallLocation: INSTALLDIR="<INSTALLPATH>"
+UpgradeBehavior: install
+ReleaseDate: 2023-10-05
```
Comment (stephengillie) [inline manifests/o/ownCloud/ownCloudDesktop/5.0.0.12278/ownCloud.ownCloudDesktop.installer.yaml:11]:
```suggestion
ReleaseDate: 2023-10-05
InstallerSuccessCodes: 3010
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**